spec/index.bs

```diff
@@ -1045,6 +1045,21 @@ a better limit is not specified.
     <tr class=row-continuation><td colspan=4>
         The maximum {{GPUBufferBinding}}.{{GPUBufferBinding/size}} for bindings for which the
         [$layout entry binding type$] is {{GPUBufferBindingType/"storage"}} or {{GPUBufferBindingType/"readonly-storage"}}.
+
+    <tr><td><dfn>maxVertexBuffers</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>8
+    <tr class=row-continuation><td colspan=4>
+        The maximum number of {{GPUVertexStateDescriptor/vertexBuffers}} when creating a {{GPURenderPipeline}}.
+
+    <tr><td><dfn>maxVertexAttributes</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>16
+    <tr class=row-continuation><td colspan=4>
+        The maximum number of {{GPUVertexBufferLayoutDescriptor/attributes}} when creating a {{GPURenderPipeline}}.
```

Comment:
this should say "in total across  {{GPUVertexStateDescriptor/vertexBuffers}}" somewhere